README.md

```diff
@@ -44,6 +44,8 @@ latest
   - To retrieve tags of private image, pass [API access token](http://docs.quay.io/api/) to `QUAYIO_TOKEN` environement variable.
 - [Docker Hub](https://hub.docker.com)
   - To retrieve tags of private image, set `DOCKER_USERNAME` and `DOCKER_PASSWORD` for [Docker Hub](hub.docker.com). If they're not set, only tags for public images will be fetched.
+- [Amazon ECR](https://aws.amazon.com/ecr/)
+  - To retrieve tags of private image, set proper `AWS_PROFILE` and `AWS_REGION` variable in your environment variables. If `AWS_PROFILE` is not set, it will use `AWS_DEFAULT_PROFILE`.
```

Comment:
`aws-sdk-go` detects and uses not only `AWS_PROFILE` (e.g., `AWS_ACCESS_KEY_ID`).

```suggestion
  - To retrieve tags of private image, [configure the profile for AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
```

Reply:
added your suggestion in e8206e9